extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/taskadapter/PodTemplateTaskAdapter.java

```diff
@@ -224,7 +224,11 @@ private Collection<EnvVar> getEnv(Task task)
             .withValueFrom(new EnvVarSourceBuilder().withFieldRef(new ObjectFieldSelector(
                 null,
                 StringUtils.format("metadata.annotations['%s']", DruidK8sConstants.TASK)
-            )).build()).build()
+            )).build()).build(),
+        new EnvVarBuilder()
```

Comment:
My understanding is we what to do things similar to [this](https://github.com/apache/druid/blob/master/extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/taskadapter/K8sTaskAdapter.java#L402-L405)
Is setting an env variable enough?

Reply:
this needs to be passed to the java druid internal peon command as a flag, and in the PodTemplateTaskAdapter, we don't actually control how the user configures that command.

But right now there's no way to even see whether the flag should be set, since we set the environment variable here, the user can read it and choose to set --loadBroadcastSegments